src/main/java/org/Olivia/Olivia.java

```diff
@@ -0,0 +1,36 @@
+package org.Olivia;
+
+import org.Olivia.IO.FileHandler;
+import org.Olivia.IO.UiHandler;
+import org.Olivia.calendar.Calendar;
+
+public class Olivia {
+    public static void main(String[] args) {
+        //each worker is attached to a calendar table to work on and an IO_handler to handle IO
+        //this design would allow the program to handle multi-user with multi-input-source case (with multi-thread)
+        //the IO_handler can be binded to a specific InputStream and a specific PrintStream (default to be stdin & stdout)
```

Comment:
What does the IO_handler here refer to? I understand it's probably `UiHandler`, perhaps consider making this clearer?

Reply:
Sorry, I forgot to change the occurrences in the comments. That is indeed UiHandler. In the previous versions it was name IO_handler.